erpnext/manufacturing/doctype/bom/bom.js

```diff
@@ -325,6 +325,7 @@ frappe.ui.form.on("BOM", {
 			freeze: true,
 			args: {
 				update_parent: true,
+				save:false,
```

Comment:
This will break the "Update cost' after submit as it won't commit the change. 

https://github.com/frappe/erpnext/blob/4f1f039145f401f6b7f601dda7855730d445379c/erpnext/manufacturing/doctype/bom/bom.py#L313-L314

Reply:
Fixed